invenio_app_ils/vocabularies/data/acq_currency.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "key": "CHF",
+    "type": "acq_currency",
+    "text": "Swiss Franc",
+    "data": { "symbol": "CHF" }
```

Comment:
why do we need the symbol here?

Reply:
I removed all the symbols, since we do not use them. That is the official symbol of Francs.